.github/workflows/main.yml

```diff
@@ -18,12 +18,18 @@ jobs:
         id: filter
         with:
           filters: |
-            crates/youki: crates/youki/**
-            crates/libcontainer: crates/libcontainer/**
-            crates/libcgroups: crates/libcgroups/**
-            tests/rust-integration-tests/runtimetest: ./tests/rust-integration-tests/runtimetest
-            tests/rust-integration-tests/integration_test: ./tests/rust-integration-tests/integration_test
-            tests/rust-integration-tests/test_framework: ./tests/rust-integration-tests/test_framework
+            crates/youki: 
+              - 'crates/youki/**'
+            crates/libcontainer: 
+              - 'crates/libcontainer/**'
+            crates/libcgroups: 
+              - 'crates/libcgroups/**'
+            tests/rust-integration-tests/runtimetest: 
+              - './tests/rust-integration-tests/runtimetest/**'
+            tests/rust-integration-tests/integration_test: 
+              - './tests/rust-integration-tests/integration_test'
+            tests/rust-integration-tests/test_framework: 
+              - './tests/rust-integration-tests/test_framework'
```

Comment:
Hey, should we add `/**` to these as well?

Reply:
Yes, I was trying this change with #1643 to see if the rules gets triggered, since this PR contains changes to the unit test file. Since #1643 included the change first, I am letting that PR to merge this change. Then I can rebase this PR. Sorry for the confusion. This PR only fixes the clippy issue and all CI related change should be made in #1643

Reply:
Hey, great :+1:
Also, nice to see you back :smile: